### PR TITLE
Adds journalist role in seed db.

### PR DESCRIPTION
**Pull request by yasmineezequiel**
## PR Content

- Role of journalist for articles in database seed

### CODE REVIEW

**Review by miyeaier — APPROVED** [review]

File: db/seeds.rb [hunk 1]
```diff
@@ -3,5 +3,5 @@
 user1 = User.create(email: 'user@mail.com', password: 'password', city: 'Stockholm', country: 'Sweden', role: 'subscriber')
 user2 = User.create(email: 'user2@mail.com', password: 'password', city: 'Stockholm', country: 'Sweden', role: 'journalist')
 user3 = User.create(email: 'user3@mail.com', password: 'password', city: 'Stockholm', country: 'Sweden', role: 'editor')
-article1 = Article.create(title: 'Swedens economy has never been better', content: 'Lorem impusm lalalala Trump', author: 'Pia Persson')
-article2 = Article.create(title: 'Brazil is building new rainforest', content: 'All of the content', author: 'Yasmine Persson')
\ No newline at end of file
+article1 = Article.create(title: 'Swedens economy has never been better', content: 'Lorem impusm lalalala Trump', author: 'Pia Persson', role: 'journalist')
```
Comment (piavW) [inline db/seeds.rb]: We need to add the `journalist_id: 2` to the articles

File: db/schema.rb [hunk 2]
```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_11_03_140639) do
+ActiveRecord::Schema.define(version: 2019_10_30_132203) do
```
Comment (GergKllai1) [inline db/schema.rb:13]: Why is the schema version older after the change?

**Review by GergKllai1 — CHANGES_REQUESTED** [review]

File: db/schema.rb (hunk 2 above)
Comment (yasmineezequiel) [reply db/schema.rb:13]: That is a great question but I was told never to touch the schema file...

Comment (yasmineezequiel) [reply db/schema.rb:13]: I tried to do a rails db:environment:set db:drop db:create db:schema:load RAILS_ENV=test to update the schema date but it has not changed and I saw on the internet that this is a common issue with schema.

Comment (GergKllai1) [reply db/schema.rb:13]: it's allright, are your tests running green?

Comment (DIzosimov) [reply db/schema.rb:13]: yes sir :)

**Review by GergKllai1 — APPROVED** [review]

**Pull request merged**